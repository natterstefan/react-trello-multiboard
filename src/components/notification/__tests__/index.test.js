import React from 'react'
import { shallow } from 'enzyme'

import Notification from '../'

describe('Component/Notification', () => {
  const props = {
    message: 'Test Message',
  }

  test('should render without throwing an error', () => {
    expect(shallow(<Notification {...props} />).dive()).toMatchSnapshot()
  })
})
