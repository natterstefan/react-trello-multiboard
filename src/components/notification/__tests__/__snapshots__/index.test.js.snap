// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Notification should render without throwing an error 1`] = `
<div>
  <WithStyles(Snackbar)
    ContentProps={
      Object {
        "aria-describedby": "message-id",
      }
    }
    action={
      Array [
        <WithStyles(IconButton)
          aria-label="Close"
          className="Notification-close-1"
          color="inherit"
          onClick={[Function]}
        >
          <pure(Close) />
        </WithStyles(IconButton)>,
      ]
    }
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "bottom",
      }
    }
    autoHideDuration={10000}
    message={
      <span
        id="message-id"
      >
        Test Message
      </span>
    }
    onClose={[Function]}
    open={true}
  />
</div>
`;
